mappings/net/minecraft/text/Text.mapping

```diff
@@ -42,6 +42,10 @@ CLASS net/minecraft/class_2561 net/minecraft/text/Text
 		COMMENT <p>A shallow copy is made for the siblings.</p>
 	METHOD method_27662 copy ()Lnet/minecraft/class_5250;
 		COMMENT Copies the text itself, excluding the styles or siblings.
+	METHOD method_30163 fromString (Ljava/lang/String;)Lnet/minecraft/class_2561;
+		COMMENT Creates a {@link Text} instance from a nullable string.
+		COMMENT @return the {@link Text} instance created from {@code string}
+		COMMENT @param string a nullable string.
```

Comment:
```suggestion
		COMMENT Creates a literal text with the given string as content.
```

Reply:
Do you describe interface methods based on their default behavior? If a subclass overrides this, the comment will be inaccurate.

Reply:
Fyi this method is static. Procyon is known for displaying interface static as defaults.

Also side note (unrelated): this is client only